module/organisations/helper.js

```diff
@@ -41,27 +42,11 @@ module.exports = class OrganisationsHelper {
                     }
                     if (roles.includes(constants.common.PLATFROM_ADMIN_ROLE)) {
 
-                        // let organisationsDoc = await cassandraDatabase.models.organisation.findAsync({},
-                        //     { raw: true, select: ["orgname", "id"] });
-
-
-
-                        // if (organisationsDoc) {
-                        //     await Promise.all(organisationsDoc.map(function (orgInfo) {
-
-                        //         let orgDetails = {
-                        //             label: orgInfo.orgname,
-                        //             value: orgInfo.id
-                        //         }
-                        //         organisationsList.push(orgDetails);
-                        //     }));
-                        // }
-
                         let request = {
                             "filters": {
                             }
                         }
-                        let organisationList = await sunBirdService.searchOrganisation(request, token);
+                        let organisationList = await sunbirdService.searchOrganisation(request, token);
```

Comment:
@rakeshSgr - We should store org info like name, id, etc in memory, basically, that's set when the service starts and updated when org is added via the admin API